components/autocomplete/overview.md

```diff
@@ -128,6 +124,37 @@ The AutoComplete provides the following popup settings:
 
 @[template](/_contentTemplates/common/get-model-from-dropdowns.md#get-model-from-dropdowns)
 
+## AutoComplete Reference and Methods
+
+The AutoComplete is a generic component and its type is determined by the type of the model you use as its data source. You can find examples in the [Data Bind - Considerations]({%slug autocomplete-databind%}#considerations) article.
```

Comment:
We have common documentation about this (https://docs.telerik.com/blazor-ui/common-features/data-binding/overview#data-type) and it will be good to optimize the content to point to it, and clean up everywhere else (not required to happen now).